tests/unit/sort.tcl

```diff
@@ -34,10 +34,22 @@ start_server {
         set _ $result
     }
 
+    proc check_sort_store_encoding {key} {
+        set listpack_max_size [lindex [r config get list-max-ziplist-size] 1]
+
+        # When the length or size of quicklist is below the threshold
+        # of 0.5(QUICKLIST_CONVERT_THRESHOLD), it will be converted to listpack.
+        if {[r llen $key] <= [expr {$listpack_max_size * 0.5}]} {
+            assert_encoding listpack $key
+        } else {
+            assert_encoding quicklist $key
+        }
+    }
+
     foreach {num cmd enc title} {
-        16 lpush quicklist "Old Ziplist"
-        1000 lpush quicklist "Old Linked list"
-        10000 lpush quicklist "Old Big Linked list"
+        16 lpush listpack "Listpack"
+        1000 lpush quicklist "Quicklist"
+        10000 lpush quicklist "Quicklist"
```

Comment:
```suggestion
        10000 lpush quicklist "Big quicklist"
```

Some distinction between the different cases can be good. The title appears in the name of test cases which is printed out when running the tests.

Reply:
Done